plugin/build.gradle

```diff
@@ -23,13 +20,13 @@ sourceSets {
 }
 
 dependencies {
-    compile gradleApi()
+    compile libraries.gradleApi
 
-    testCompile gradleTestKit()
-    testCompile 'junit:junit:4.12'
-    testCompile 'com.google.truth:truth:0.30'
-    testCompile 'com.google.guava:guava:19.0'
-    testCompile 'com.google.code.findbugs:jsr305:3.0.0'
+    testCompile libraries.gradleTestKit
+    testCompile libraries.junit
+    testCompile libraries.truth
+    testCompile libraries.guava
+    testCompile libraries.findbugs.annotations
```

Comment:
how comes this one would be like this vs. just `libraries.findbugs`?

I could see it for android support libs where there's several that fit under a single group but didn't follow the reasoning here.